packages/hydra-cli/test/helpers/JsonFieldRenderer.test.ts

```diff
@@ -0,0 +1,61 @@
+import { expect } from 'chai'
+import * as fs from 'fs-extra'
+
+import { WarthogModel } from '../../src/model'
+import { ModelRenderer } from '../../src/generate/ModelRenderer'
+import { JsonFieldRenderer } from '../../src/generate/JsonFieldRenderer'
+import { WarthogModelBuilder } from '../../src/parse/WarthogModelBuilder'
+
+describe('JsonFieldRenderer', () => {
+  let warthogModel: WarthogModel
+  let jsonFieldTemplate: string
+  let modelTemplate: string
+
+  before(() => {
+    jsonFieldTemplate = fs.readFileSync(
+      './src/templates/jsonfields/jsonfields.model.ts.mst',
+      'utf-8'
+    )
+    modelTemplate = fs.readFileSync(
+      './src/templates/entities/model.ts.mst',
+      'utf-8'
+    )
+
+    warthogModel = new WarthogModelBuilder(
+      'test/fixtures/jsonfields.graphql'
+    ).buildWarthogModel()
+  })
+
+  it('shoud render typed json object', () => {
+    const rendered = new JsonFieldRenderer(warthogModel).render(
+      jsonFieldTemplate
+    )
+
+    expect(rendered).include(
+      `@InputType('EventParamInput')\n@ObjectType()\nexport class EventParam {`,
+      'shoud have class defination with decorators'
+    )
+  })
+  it('shoud render array typed json object', () => {
+    const rendered = new JsonFieldRenderer(warthogModel).render(
+      jsonFieldTemplate
+    )
+
+    expect(rendered).include(
+      `@Field(() => [AdditionalData])\n  additionalData!: AdditionalData[]`,
```

Comment:
I suggest using `compact` method from utils for comparing such big strings. It simple strips all extra new lines and spaces, so there's no need to keep track of `\n` or extra spaces. Here it would be 
`expect(compact(rendered)).include(compact(`@Field(() => [AdditionalData]) additionalData!: AdditionalData[]`))`

Apart from convenience, that should save from platfrom-specific issues, e.g. on Windows the new line is `\r\n` not `\n` and the tests gonna fail